app/_locales/en/messages.json

```diff
@@ -1035,6 +1035,21 @@
     "message": "File import not working? Click here!",
     "description": "Helps user import their account from a JSON file"
   },
+  "flaskExperimentalText1": {
+    "message": "Using Flask can greatly increase your risk of fund loss:"
+  },
+  "flaskExperimentalText2": {
+    "message": "if you use it to install non-trustworthy Snaps"
+  },
+  "flaskExperimentalText3": {
+    "message": "if you do not review confirmations before approving changes"
```

Comment:
Use sentence case
```suggestion
    "message": "If you do not review confirmations before approving changes"
```

Reply:
To @rekmarks point it's a continuation of the sentence so lowecase works here. Please ignore.